csrc/python_frontend/python_bindings.cpp

```diff
@@ -445,6 +474,17 @@ void initNvFuserPythonBindings(PyObject* module) {
       .value("shared", MemoryType::Shared)
       .value("global", MemoryType::Global);
 
+  //! Scheduler Type for scheduling
+  py::enum_<ScheduleHeuristic>(nvfuser, "SchedulerHeuristic")
+      .value("no_op", ScheduleHeuristic::NoOp)
+      .value("pointwise", ScheduleHeuristic::PointWise)
+      .value("reduction", ScheduleHeuristic::Reduction)
```

Comment:
```suggestion
      .value("matmul", ScheduleHeuristic::Matmul)
      .value("reduction", ScheduleHeuristic::Reduction)
```

Reply:
This would allow us to fuse matmuls manually even without the `NVFUSER_ENABLE=fuse_matmul` option.